### DISCUSSION TITLE
Logging feature / Current state and future

### DISCUSSION THREAD

**Discussion by ghost** [open-telemetry/opentelemetry-go#3187] [Q&A]
I'd like to ask what's the recommended approach for logging with otel, because in the main readme for this repo the state is described as frozen.
This leads to a problem somehow, because in the main golang repo there is a discussion for a proposal for structured logging and there they refer to the current work of otel...
https://github.com/golang/go/discussions/54763

So, maybe I haven't found the correct place concerning logging in otel.

Maybe someone could provide a bit of clearification.

Thank you

**Comment by dmathieu** [comment]
Hi,

Logs is indeed frozen until metrics are in a finalized enough state. We should then start looking at their implementation.
In the mean time though, you can find more details about what they should look like in the specifications: https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/logs
